README.md

```diff
@@ -1,8 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-
-
+To link to your database, create .env in be-nc-news with the following line:
```

Comment:
Good start! Remember our connection files are set up to make use of two different .env files programatically - in which case we should tell the user about them so that they are able to correctly run the repo, whether its the tests or if its the server itself
We can also point them to setup.sql so they know what to call the databases within the .env files